pkg/cmd/env/templates/history.tmpl

```diff
@@ -0,0 +1,15 @@
+{{define "bash"}}
+gardenctl target history | fzf --tac --no-sort --height 40% --layout reverse --info inline --border --preview 'echo {}' --preview-window up,1,border-horizontal | bash
```

Comment:
was there a reason why you pipe the fzf selection to the respective shell (`| bash`) instead of using the eval command? There is also an `eval-cmd` helper template

Reply:
I'm not quite sure about `eval-cmd` helper template,  removed the respective shell `| bash`

Reply:
ok after the `| zsh` was removed in my case.  meeting some weird issues. maybe the wrong way I used eval command

```
(N/A:N/A)➜  ~ gh
gardenctl target --garden local
(N/A:N/A)➜  ~ gk
(N/A:N/A)➜  ~ gp
Error: no garden cluster targeted
```

1. 'eval $(gardenctl history-env zsh)'  // It looks like not target 
then
2. 'eval $(gardenctl kubectl-env zsh)' // can't export kubeconfig either.  

now I just put `| bash`, `| zsh` , '| fish' back at the end of each line.

Reply:
you can't just remove the ` | zsh` without adding an eval. Otherwise `fzf` will just print out the selection to STDOUT
you can do something like `{{template "eval-cmd" dict "shell" .shell "cmd" "gardenctl target history | fzf --tac --no-sort --height 40% --layout reverse --info inline --border --preview 'echo {}' --preview-window up,1,border-horizontal"}}`
in the end this template wraps the command with an eval statement for the respective shell

Reply:
done, add new template expects `PowerShell` 7214aae